src/components/base/Base.js

```diff
@@ -1398,13 +1398,31 @@ export default class BaseComponent extends Component {
     return super.setInputMask(input, (inputMask || this.component.inputMask), !this.component.placeholder);
   }
 
+  addAutofillHoneyInput(container, input) {
+    const autofillInput = this.ce('input', {
+      type: 'text',
+      name: this.info.attr.name,
+      style: 'display: none',
+    });
+
+    input.addEventListener('change', (event) => {
+      autofillInput.value = JSON.stringify(event.detail.value);
+    });
+
+    container.appendChild(autofillInput);
+  }
+
   /**
    * Creates a new input element.
    * @param {HTMLElement} container - The container which should hold this new input element.
    * @returns {HTMLElement} - Either the input or the group that contains the input.
    */
   createInput(container) {
     const input = this.ce(this.info.type, this.info.attr);
+    if (this.info.type === 'select') {
+      this.addAutofillHoneyInput(container, input);
```

Comment:
Why not extend ```createInput``` in the SelectComponent class and add this so you don't have to do this?